org.adempiere.base/src/org/compiere/model/MUserDefInfo.java

```diff
@@ -81,7 +81,7 @@ public MUserDefInfo(Properties ctx, ResultSet rs, String trxName) {
 	private static MUserDefInfo[] getAll (Properties ctx, int infowindow_ID )
 	{
 		if (m_fullList == null) {
-			m_fullList = new Query(ctx, MUserDefInfo.Table_Name, "IsActive='Y'", null).list();
+			m_fullList = new Query(ctx, MUserDefInfo.Table_Name, "IsActive='Y'", null).setClient_ID().list();
```

Comment:
This is wrong, as is designed m_fullList is intended to have a list from all tenants.

To make this work would require a refactor similar to what @hengsin did in MUserDefWin here:
https://github.com/idempiere/idempiere/commit/2ab2211f81f4ecb3928a56506ddd7712731a1036#diff-18f1e200c9da97cb1ba9cb7290831a270c9dd9d5a9721ff68628a16dd1b574fd